src/Apps/Order/Components/__tests__/AddressVerificationFlow.jest.tsx

```diff
@@ -0,0 +1,38 @@
+import { screen } from "@testing-library/react"
+import { AddressVerificationFlowFragmentContainer } from "Apps/Order/Components/AddressVerificationFlow"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+const { renderWithRelay } = setupTestWrapperTL({
+  Component: AddressVerificationFlowFragmentContainer,
+  query: graphql`
+    query AddressVerificationFlow_Test_Query($address: AddressInput!)
+      @relay_test_operation {
+      verifyAddress(address: $address) {
+        ...AddressVerificationFlow_verifyAddress
+      }
+    }
+  `,
+})
+
+describe("AddressVerificationFlow", () => {
+  it(`renders the "Check your delivery address" variant when verificaiton is "NOT_FOUND"`, async () => {
```

Comment:
**nitpick [blocking]:** 

There is a typo in the test description: "verificaiton" should be "verification"

*format via [Conventional Comments](https://conventionalcomments.org/)*